### ISSUE TITLE
[Suggestion] Restructuring documentation

### ISSUE THREAD

**Issue by miphilomath** [pythonindia/junction#697]
Since Junction's documentation is distributed at various places and requires restructuring, I would like to propose the following structure for the readthedocs documentation, suggesting appropriate location for the existing documentation on GitHub wiki. Creating a structure before writing pages will help us write individual pages and recollect them easily.
I suggest to have 4 level of documentation:

----
1. **Welcome Page**
  - With table of contents reference for whole project for easier and efficient navigation.
  - Link to the "docs" labelled issue page to welcome more documentation contribution.
  - License at the end (optional)

2. **User Level,** to welcome more use of the project among speakers and by other conference organizers with following sections/pages.
  - Installation Guide for local and production environment for different platforms.
  - Quick Start / Getting Started guide, which will cover the steps after installation to use the project. A brief on how one can use the project to setup their CFP acceptance system.
  - Features the project offers
  - How-To(s) * o setup and use the project as described [here](https://github.com/pythonindia/junction/issues/627) (migrate content from existing Github wiki over here)
  - FAQ section
  - [Best Practices](https://github.com/pythonindia/junction/wiki/Speaker-best-practices) and [speaker guidelines](https://github.com/pythonindia/junction/wiki/Speaker-Guidelines)
  - Link to release notes page for the sake of history.

3. **Contribution** page
  -  Contribution guidelines with best practices to create a patch
  - License declaration
  - Pre-requisites such as git/python and django and sharing relevant resources
  - Redirection to easy-to-fix label issues on github
  - Redirection to Development/developer page

4. **Development Page**, for developers working on the project
  - Link to contribution guidelines
  - Getting started page link / Installation Page link
  - API resources
  - Structure of the project, detailing on the individual application
----

**Comment by miphilomath** [comment]
![junction](https://user-images.githubusercontent.com/26297035/84573268-02601000-ad8f-11ea-8ff0-0a0061d9de07.png)

**Comment by palnabarun** [comment]
@miphilomath -- Thank you for kicking this off. All of the above plans look good to me other than the following comment,
How about removing the Developer Guide? API Reference and Project Structure are the proposed sections adding value. Project Structure can be documented in the Contributor Guide itself and API Reference can be a separate top-level section.

**Comment by miphilomath** [comment]
Yeah, I guess we can keep the contents of developer guide in contribution guide. I kept a separate developer page assuming that the page will grow bigger with details of Frontend and Backend components and applications detail later. But I guess, we can keep them in contribution guide until that happens.

Also, should I go ahead with the Pull request for the documentation as suggested above?
